### PR TITLE
[FRONT-142] Support filtering by combined metrics

### PR DESCRIPTION
**Pull request by WrappedSuperClass**
## Description of Issue
The forksPerContr and the issuesPerContributer are calculated in the frontend. Remove the calculation and use instead the newly added attributes provided from the backend
## How I implemented
Edited the query, as i'm using windows, i had to change the script in codegen.config.ts, but the changes were not commited and/or pushed. 

With the new props of project i replaced the calculated one in the right sidebar and in the table. 

I added the new attributes as filter and sorting options, and replaced the parseInt with parseFloat to make the filter work for the new attributes too

### CODE REVIEW

File: src/components/page/details/RightSidebar.tsx [hunk 1]
```diff
@@ -43,11 +43,11 @@ const RightSidebar = ({ project }: Props) => {
         <Sidebar.Box.GithubStatItem Icon={RepoForkedIcon} value={project.forkCount as number} />
         <Sidebar.Box.GithubStatItem
```
Comment (ArnoClaude) [inline src/components/page/details/RightSidebar.tsx:44]: image
Contributors should be 0, not empty.
Also, can you re-order the icons such that it's either Issues, Issues/Contrib., Forks, Forks/Contrib.?

Comment (ArnoClaude) [inline src/components/page/details/RightSidebar.tsx:44]: image
Contributors should be 0, not empty.
Also, can you re-order the icons such that it's Issues, Issues/Contrib., Forks, Forks/Contrib.?

**Review by ArnoClaude — CHANGES_REQUESTED** [review]
Great job Lukas! I just left two small remarks :)

File: src/components/page/details/RightSidebar.tsx (hunk 1 above)
Comment (kigadye) [reply src/components/page/details/RightSidebar.tsx:41]: done

**Review by kigadye — APPROVED** [review]

**Review by pczern — APPROVED** [review]
needed for handover tomorrow

**Pull request merged**